wger/weight/urls.py

```diff
@@ -44,10 +44,10 @@
     # url(r'^overview/$',
     #     views.overview,
     #     name='overview'),
-    url(r'^api/get_weight_data/(?P<username>[\w.@+-]+)$', # JS
```

Comment:
@Emmanuel-Dominic a little enlightenment about the importance of modifying the routes from `api` to `v2`

Reply:
This was not redirected to the main URL file, so had to reformat it and use it in my implementation.